node/node_test.go

```diff
@@ -1456,6 +1463,478 @@ func TestNode_StartConsensusGenesisBlockNotInitializedShouldErr(t *testing.T) {
 
 }
 
+func TestStartConsensus_NilSyncTimer(t *testing.T) {
```

Comment:
There are some tests missing from this package:
ApplyOptions(): node is running should err
createConsensusTopic(): has a  bunch of if conditions that are not tested
GetCurrentPublicKey(): is not tested at all
ValidatorStatisticsApi(): is not tested at all

Reply:
done.